patches/0017-mali-Avoid-splitting-lines-across-multiple-printf-calls.patch

```diff
@@ -0,0 +1,52 @@
+mali: Avoid splitting lines across multiple printf calls
+
+Prevents single lines from being shown as two lines in dmesg output with
+newer Linux kernels.
+
+Signed-off-by: Jonathan Liu <net147@gmail.com>
```

Comment:
Some more details about the issue would be nice here.

Reply:
I have added more details about the issue.